common/arrays/arrays.ts

```diff
@@ -1,5 +1,7 @@
 export function flatten<T>(nestedArray: T[][]) {
```

Comment:
we shouldn't even need this function any more, we can use `Array.flat()` now.

Reply:
Doesn't seem to be available? Isn't in any of our code base too.

Reply:
It's available, it's just that the open-source `tsconfig.json` doesn't include an up-to-date lib. If you change the relevant line in `tsconfig.json` to `"lib": ["es2019", "dom"]` (which would then match what we have in closed-source, where we use e.g. `flatMap`), then you can use it.